### PR TITLE
pl-PL: Updated polish translations

### PR DESCRIPTION
**Pull request by marcinkunert**
Applying for issue(s):
- #1821
- #1826 
- #1847 
- #1844
- #1850
- #1862
- #1866
- #1876
- #1887
- #1888
- #1903

### CODE REVIEW

File: data/language/pl-PL.txt [hunk 1]
```diff
@@ -728,10 +723,10 @@ STR_1339    :{BLACK}Oczekiwanie na rezultaty testu...
 STR_1340    :{WINDOW_COLOUR_2}Maks. prędkość: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Czas przejazdu: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} / 
+STR_1343    :{DURATION} /
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:726]: Do not "fix" the trailing whitespace in this file - it is supposed to be there.

File: data/language/pl-PL.txt [hunk 2]
```diff
@@ -1958,6 +1953,7 @@ STR_2741    :RollerCoaster Tycoon 2
 STR_2742    :Nie znaleziono css50.dat
 STR_2743    :Przekopiuj data\css17.dat z folderu RCT1 do data\css50.dat folderu instalacji RCT2.
 STR_2749    :Mój nowy scenariusz
+# New strings used in the cheats window previously these were ???
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:1956]: Please do not copy the comments from the en-GB file.

File: data/language/pl-PL.txt [hunk 3]
```diff
@@ -3550,7 +3517,123 @@ STR_6258    :Clear (transparent)
 STR_6259    :Wyłączone
 STR_6260    :Show blocked tiles
 STR_6261    :Show wide paths
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:3519]: Two untranslated strings here.

File: data/language/pl-PL.txt [hunk 4]
```diff
@@ -3406,21 +3381,13 @@ STR_6100    :Rozłączyłeś się z serwerem.
 STR_6101    :Wartość atrakcji nie spada wraz z upływem czasu.
 STR_6102    :{SMALLFONT}{BLACK}Wartość atrakcji nie będzie spadał z czasem, dlatego klienci nie będą po czasie stwierdzić że jest za droga.
 STR_6103    :{SMALLFONT}{BLACK}Opcja ta nie jest dostępna w grze wieloosobowej.
-STR_6104    :Corkscrew Roller Coaster
 STR_6105    :Hypercoaster
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:3384]: Do the ride names here not need translation?

File: data/language/pl-PL.txt (hunk 1 above)
Comment (marcinkunert) [reply data/language/pl-PL.txt:726]: You are probably right. I've compared this to en_GB and there is also no trailing whitespace. I should have first tried to find this in the game and then decide about the translation, not just guessing. I'll undo this.

File: data/language/pl-PL.txt (hunk 2 above)
Comment (marcinkunert) [reply data/language/pl-PL.txt:1956]: The comment is kinda useful to make the comparing of two translation files easier. If the total line number count matches that means that no translations are missing. I've translated this comment, although leaving the original comment would be more useful in my opinion.

File: data/language/pl-PL.txt (hunk 3 above)
Comment (marcinkunert) [reply data/language/pl-PL.txt:3519]: Translated. So far I just wanted to start with the translation process. There are still a few missing. I'll try to find some time to update them all.

File: data/language/pl-PL.txt (hunk 4 above)
Comment (marcinkunert) [reply data/language/pl-PL.txt:3384]: They most likely do. So far I just wanted to start with the translation process. There are still a few missing. I'll try to find some time to update them all.

File: data/language/pl-PL.txt (hunk 1 above)
Comment (Gymnasiast) [reply data/language/pl-PL.txt:726]: > I've compared this to en_GB and there is also no trailing whitespace.

There _is_ trailing whitespace in en-GB.txt, including on this line. Your IDE might auto"fix" it. Again, please restore all whitespace "fixes" in this file.

Comment (marcinkunert) [reply data/language/pl-PL.txt:726]: My bad. Ofc I'll restore them.

Comment (marcinkunert) [reply data/language/pl-PL.txt]: Restored. For some reason IntelliJ kept removing trailing whitespaces.

**Pull request merged**